src/data/footer.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
 			value: 'NEWS',
 		},
 		{
-			href: 'http://survey.curtmfg.com/surveys/5',
+			href: 'http://warranty.curtgroup.com/surveys/5',
```

Comment:
Just realized that's broken, let me check if our certificate is wildcard or not.